package.json

```diff
@@ -113,7 +113,7 @@
         "serve:build": "http-server build/ -s -p 3001 -P \"http://127.0.0.1:3001?\"",
         "build:test": "cross-env REACT_APP_TEST_MODE=true craco build",
         "build:production": "craco build",
-        "analyze": "cross-env ANALYZE=true craco build:production",
+        "analyze": "cross-env ANALYZE=true craco build",
```

Comment:
Am I missing something or is there no build command?

Reply:
craco != yarn. Here build is literally "build the app". Not "run the build npm task"